armi/utils/tests/test_utils.py

```diff
@@ -202,6 +202,14 @@ def test_slantSplit(self):
         self.assertListEqual(x1, [1.0, 2.0, 3.0, 4.0])
         self.assertListEqual(x2, [4.0, 3.0, 2.0, 1.0])
 
+    def test_newtonsMethod(self):
```

Comment:
Good call on leaving the cs out as we are hoping to get rid of most things that need some kind of global CS in our roadmap.